### DISCUSSION TITLE
Switch from one scenario to another using a action

### DISCUSSION THREAD

**Discussion by hafbtm** [Nain57/Smart-AutoClicker#886] [General Feature Requests]
### Describe the feature

The application is already too functional today, I take this opportunity to thank the DEV, I think it would be practical if you add a function to change from one scenario to another through a action, something like bridge mode between scenarios

### Describe the usage

There is currently no way for one scenario to directly interact with another, I hope I'm not missing something.

### Is it a duplicate?

- [X] There is no duplicates

**Comment by Van493** [comment]
For now this can be done by carefully composing the separate scenarios into one big scenario, with additional Event State actions or Counters as control state.

**Comment by shmaungms** [comment]
Whilst this can be done by carefully composing separate scenarios in to one big one I can see how this would be a useful feature and keep your scenarios more organised. It's possible that most of the time you want to run them separately but only occasionally together.

**Comment by csm10495** [comment]
+1. Would use this feature to string together existing scenarios. It's possible via copying conditions/etc. but that would be kind of rough compared to a direct trigger to other scenario.

**Comment by TheUltimateAbsol** [comment]
+1. Have been trying to make a big automation for a certain horsey game but I'm starting to hit processing limitations after adding about 200 different events, and would be nice if I could process only a very certain subset at a time without having to do complicated "enable/disable" logic for everything. Definitely needed when doing complex tasks, such as counting elements between screens.
